tailwarden/komiser
# Added ELB listener rules and change listener name

## Problem

Solves #534 #535

## Solution

-  Updated code to show Listener rules
## Changes Made
- Updated listener name from Listener ARN to ELB Name because its easy to identify using the ELB name.
- Added support to display ELB Listener Rules based on the ELB Type.
## How to Test

- Make sure your AWS account has AWS Application/Network/Gateway ELB
- Build and run this branch
- Choose ELB App/Net/Gat Listener Rule as the service filter option.
- Follow step 3 but pick ELB App/Net/Gat Listeners to verify the new Listner names.

## Screenshots

![image](https://github.com/tailwarden/komiser/assets/25551553/1b2cab54-a402-4138-b255-1cf87c818dd2)
![image](https://github.com/tailwarden/komiser/assets/25551553/5928b4a3-2d1e-4995-8e97-c9aa3ca8ee2c)


## Notes
- Only tested using Application ELB. Should work for Network and Gateway too. 

## Checklist
